apps/docs/pages/guides/database/extensions/pgnet.mdx

```diff
@@ -53,11 +53,11 @@ Procedural languages are automatically installed within `pg_catalog`, so you don
 </TabPanel>
 </Tabs>
 
-## `http_get` [#http_get]
+## `http_get`
```

Comment:
@supabase/docs @supabase/frontend for some reason the custom anchor `[#http_get]` definition isn't working for `h2`s. It's working fine for the `h3`s below.

Reply:
Seems to be the code font formatting expect an extra `#` (but it still breaks the link in the table of contents):
![Screenshot 2023-01-17 at 3 46 22 PM](https://user-images.githubusercontent.com/7026076/213036892-6e5bf251-2c22-4d2a-8b78-66454f46791d.png)